_posts/2018-06-13-hackillinois-2019-mission.markdown

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: "Something old, something new…"
+date: 2018-06-20 12:00:00
+categories: preview
+tags: HackIllinois2018, HackIllinois2019, featured
+author: Kyle Begovich
+---
+
+Even though it's only June, planning for HackIllinois 2019 is already well under way, and our Summer Team would love to offer you an inside look at some of our motivations and guiding principles.
+
+# Something Old
+
+The blog has already explored the event last year in a few retrospective pieces, but here we will begin focusing on how it all applies to next year. We are currently in the process of engaging mentors from last year to get their take on their weekend on campus: what they liked and what they would like changed. These mentors are the heart of our hacker experience, tons of fun to host, and exceptionally talented contributors in their field. If you didn't get to engage with the mentors at the event, we hope to make this opportunity more central to the hacker experience this year.
```

Comment:
I definitely agree with Justin here. It would be cool to bring in some shortcomings of HackIllinois from past years and talk about how were working to improve them. Maybe even hinting to Arnav's upcoming tech posts? Unless his article goes out before this one ofc.

Reply:
If you didn't get to engage with the mentors at the event, we hope to make this opportunity more central to the hacker experience this year. -> If you didn't get to engage with the mentors at the event, don't worry, we hope to make this opportunity more central to the hacker experience this year.